poetry.lock

```diff
@@ -0,0 +1,57 @@
+[[package]]
+category = "main"
+description = "Python datetimes made easy"
+name = "pendulum"
+optional = false
+python-versions = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*"
```

Comment:
Only supporting 3.5+ . (using f-strings, etc)

Reply:
That's the versions for pendulum!